content/concept-checks.md

```diff
@@ -0,0 +1,103 @@
+# Checks
+
+In the XRP Ledger, a Check is similar to a personal paper check. Like traditional checks, XRP Ledger Checks start with the sender of the funds creating a Check that specifies an amount and receiver. The receiver cashes the check to pull the funds from the sender's account into the receiver's account. No money moves until the receiver cashes the Check. Because funds are not put on hold when the Check is created, cashing a Check can fail if the sender doesn't have enough funds when the receiver tries to cash it, just like traditional checks. If there's a failure cashing the check, the sender can retry until the check expires.
```

Comment:
I think it may best serve the reader to begin with a description or definition, if possible, before moving onto analogies. Something like the following, but better:

> In the XRP Ledger, Checks are payments that allow senders to initiate either XRP or issued currency payments that earmark funds, and receivers to choose if and when they accept those payments. A Check is similar to a personal paper check...

Reply:
Revised to: 

> The Checks feature in the XRP Ledger allows users to create deferred payments that can be canceled or cashed by the intended recipients. Like personal paper checks, XRP Ledger Checks start